server/models/userModel.js

```diff
@@ -1,20 +1,23 @@
 import mongoose from 'mongoose';
 import bcrypt from 'bcryptjs';
 
-const UserSchema = new mongoose.Schema({
-  name: {
-    type: String,
+const UserSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    password: {
+      type: String,
+      required: true,
+    },
   },
```

Comment:
Try not to refactor for indentation. It will save your name in git regarding who was the last one to change that line and in git blame, your name would come. So it would be difficult to know the actual person to make changes and reason for making it if we are stuck in an issue.

Reply:
Ok, I will keep in mind. :ok_hand: